WeatherStationClient.py

```diff
@@ -8,6 +9,28 @@
 from Crypto.Hash import SHA256
 
 
+def get_nested_attr(obj, attr_name):
```

Comment:
I never use it but as far i know something similar exists. Its called jsony query?

Reply:
Może i jest ale nic mi o tym nie wiadomo. Chętnie się dowiem.

Czy to zewnętrzna biblioteka? Jeśli tak, to miało takich nie być.

> **jsony** query

No ale ja tutaj nie chcę obrabiać JSONa tylko pythonowy obiekt.